### PR TITLE
Add Extension Marketplace

### PR DESCRIPTION
**Pull request by MUzairS15**
Signed-off-by: MUzairS15 <muzair.shaikh810@gmail.com>

**Description**

This PR adds "Extension Marketplace" as a Menu Item in Navigator
Screenshot 2022-08-12 at 5 21 13 PM
Screenshot 2022-08-12 at 5 23 11 PM

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/pages/extensions.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  },
+  img : {
+    marginLeft : "0.5rem",
+    height : 'auto',
+    width : "auto",
+    maxWidth : "100px",
+    maxHeight : "100px"
+  },
+  content : {
+    textAlign : "right",
```
Comment (leecalcote) [inline ui/pages/extensions.js:49]:
```suggestion
    textAlign : "left",
```

File: ui/components/Navigator.js [hunk 2]
```diff
@@ -473,6 +473,15 @@ const categories = [
       },
     ],
   },
+  {
+    id : "Extension",
+    // icon: <ConformanceIcon style={drawerIconsStyle} />,
+    // hovericon : <ConformanceHover style={drawerIconsStyle} />,
```
Comment (Chadha93) [inline ui/components/Navigator.js:479]: We need an icon here to show in the navigator.

**Review by Chadha93 — COMMENTED** [review]
Are we adding this page under SMI conformance? As a sub-menu?

File: ui/components/Navigator.js (hunk 2 above)
Comment (leecalcote) [reply ui/components/Navigator.js:479]: here we are - meshery-extensions

File: ui/pages/extensions.js (hunk 1 above)
Comment (Chadha93) [reply ui/pages/extensions.js:49]: This won't match the desired Figma design, It will bring the Text above the icon.

**Comment by Chadha93** [comment]
@leecalcote @MUzairS15 Are we adding this page under SMI conformance? As a sub-menu?

**Comment by MUzairS15** [comment]
> @leecalcote @MUzairS15 Are we adding this page under SMI conformance? As a sub-menu?

No, A separate menu item

**Comment by Chadha93** [comment]
> > @leecalcote @MUzairS15 Are we adding this page under SMI conformance? As a sub-menu?
> 
> No, A separate menu item

image

**Comment by MUzairS15** [comment]
> > > @leecalcote @MUzairS15 Are we adding this page under SMI conformance? As a sub-menu?
> > 
> > 
> > No, A separate menu item
> 
> image

Actually, there is no image, that's why it looks like it is a sub-item of `SMI-Conformance`, I added the icon.Pls check

**Review by Chadha93 — APPROVED** [review]
LGTM

**Comment by leecalcote** [comment]
Thank you, gents.

**Comment by leecalcote** [comment]
@Chadha93 you might have missed this suggestion prior to merging - https://github.com/meshery/meshery/pull/6014/files#r944398454

**Pull request merged**